src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/FlatFWTest.java

```diff
@@ -61,237 +68,267 @@ public void shouldProvideTypeId() throws Exception
     @Test
     public void shouldDefaultValues() throws Exception
     {
-        int limit = flatRW.wrap(buffer, 0, 100)
-                .fixed1(10)
-                .string1("value1")
-                .string2("value2")
-                .build()
-                .limit();
+        int limit = setAllRequiredValues(flatRW.wrap(buffer, 0, 100));
         flatRO.wrap(buffer,  0,  limit);
         assertEquals(222, flatRO.fixed2());
         assertEquals(333, flatRO.fixed3());
     }
 
-    @Test(expected = IndexOutOfBoundsException.class)
-    public void shouldFailToSetFixed1WithInsufficientSpace()
+
+    @Test
+    public void shouldSetAllValues() throws Exception
     {
-        flatRW.wrap(buffer, 10, 10)
-               .fixed1(10);
+        setAllValues(flatRW.wrap(buffer,  0,  buffer.capacity()));
+        flatRO.wrap(buffer,  0,  100);
```

Comment:
Again, this is not the right approach, you are mixing RW and RO so will not catch bugs where the actual bytes set in memory are wrong (i.e. both methods, setter and accessor, are wrong). Follow the approach in FlatWithOctetsFWTest, by having a setAllValues(MutableDirectBuffer, offset) method which sets up the expected bytes into the expected buffer, then comparing buffer (as written to by flatRW) with expected.